src/main/java/Duke.java

```diff
@@ -1,10 +1,293 @@
+import duke.exception.MissingDateException;
+import duke.exception.MissingDescriptionException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
+
+
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
 public class Duke {
+
+    protected static ArrayList<Task> tasks = new ArrayList<>();
+
+    private static void parseInput(Scanner in) {
+        String line;
+        System.out.println("Storing Up to 100 Tasks");
+        do {
+            System.out.println("Awaiting your command");
+            line = in.nextLine();
+            String[] words = line.split(" ", 2);
+            String command = words[0];
+
+            switch (command) {
+            case "list":
+                printTasks();
+                break;
+            case "mark":
+                markTask(words);
+                break;
+            case "unmark":
+                unmarkTask(words);
+                break;
+            case "todo":
+                addToDo(words);
+                break;
+            case "deadline":
+                addDeadline(words);
+                break;
+            case "event":
+                addEvent(words);
+                break;
+            case "delete":
+                deleteTask(words);
+                break;
+            case "help":
+                printHelp();
+                break;
+            default:
+                if (!line.equalsIgnoreCase("bye")) {
+                    System.out.println("Sorry, I did not understand that command. Input help to find out more.");
+                }
+                break;
```

Comment:
You might want to just create another `case` to handle `"bye"`. The good practice for `default` is usually to handle unregistered commands. This is also to improve SLAP.